package.json

```diff
@@ -62,7 +62,7 @@
     "electron-log": "^4.3.5",
     "electron-serve": "^1.0.0",
     "electron-updater": "^4.3.4",
-    "expensify-common": "git://github.com/Expensify/expensify-common.git#b5d9167a26cfd40beebdfb79958b3bbed1af7b64",
+    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#b5d9167a26cfd40beebdfb79958b3bbed1af7b64",
```

Comment:
Should we also update the `react-native-onyx` url to match the same convention?

Reply:
Reverted this back to https because ssh encountered a publickey error